app/helpers/embed_player_helper.rb

```diff
@@ -47,11 +66,35 @@ def embed_player_episode_iframe(ep, type = nil, preview = false)
     end
   end
 
+  def embed_player_podcast_iframe(podcast, options, preview = false)
+    src = embed_player_podcast_url(podcast, options, preview)
```

Comment:
completely copied over from `embed_player_episode_iframe` except to change this `src` attribute.